### DISCUSSION TITLE
Block app launch during uninstall

### DISCUSSION THREAD

**Discussion by EvgenPervenenko** [wixtoolset/issues#6956] [Questions]
Hello, all, are there a way to block app launch during uninstall using wixtoolset?

There is a time when user can run application during uninstall, it causes errors ( application can't find some files which already moved in temp directory )

**Comment by robmen** [comment]
A custom bundle application could create some sort of mutex and hold on to it for the extent of the uninstall. The app would have to look for that. There isn't a great way to do that with just the Windows Installer.
